source/documentation/runbooks/015-dashboard.html.md.erb

```diff
@@ -0,0 +1,47 @@
+---
+owner_slack: "#data-catalogue"
+title: Metadata dashboard
+last_reviewed_on: 2024-12-11
+review_in: 3 months
+---
+# <%= current_page.data.title %>
+
+The metadata dashboard is used internally within the team to track completeness of the metadata shown in Find MoJ data. It's implemented in [AWS QuickSight](https://docs.aws.amazon.com/quicksight/latest/user/welcome.html).
+
+To see it, log into the [Analytical Platform Control Panel](https://controlpanel.services.analytical-platform.service.justice.gov.uk/) and
+click on QuickSight.
+
+Under Dashboards, select "FMD Dashboard v1.4" (or the latest version).
+
+## Add a new user
+To grant a new team member access to QuickSight in the Analytical Platform:
```

Comment:
While this link will become stale at some point I imagine the AP will want to add the user via this issue rather than have people using EntraID directly https://github.com/ministryofjustice/analytical-platform/issues/new?template=quicksight-trial-form.yaml

Reply:
@julialawrence, can you confirm whether the above link is what we should be using in the future? (Last time we didn't fill this out, and I added everyone to the entra group).

Just want to make sure we understand the process for if new people join the team.